### DISCUSSION TITLE
so-allow take a long time to add any IP

### DISCUSSION THREAD

**Discussion by mahmoudelhaj** [Security-Onion-Solutions/securityonion#9741] [General] [answered]
so-allow take a long time to add any IP
The process takes about 10 minutes

![image](https://user-images.githubusercontent.com/29997801/217540829-2682a32e-c5f5-4881-804b-f65cdc8f9344.png)

**Comment by InfosecGoon** [comment, marked as answer]
That's unusual, but if your system is already applying a salt highstate or something it's certainly possible. Would it be more efficient to add a subnet rather than individual IPs? So-allow will accept CIDR ranges as well.
